code-challenges.js

```diff
@@ -43,9 +91,33 @@ var myMapNumbers2 = [8, -7, 0, 6, 12]
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
-
-
-
+// Create a function that takes in 1 argument, which is a string
+const noVowels = (str) => {
+    // Convert the string into an array using '.split()' and store it in a variable called 'lettersArr'
+    let lettersArr = str.split('')
+    // Use '.filter()' to check if the element is a vowel
+    let consonantsArr = lettersArr.filter(value => {
+        // '.filter' requires a true or false to filter the array
+        // so we need if/else conditionals
+        // Check if an array containing the vowels includes the value
+        // Force value to lowercase to capture those that are capitalized as well
+        if ( !['a','e','i','o','u'].includes(value.toLowerCase()) ) {
```

Comment:
`.includes()` returns `true` or `false` already so you actually don't need to create a whole if-else statement  

```suggestion
        .filter(value => !['a','e','i','o','u'].includes(value.toLowerCase()));
```

Reply:
awesome use of built-in methods!